### DISCUSSION TITLE
We havea problem to sending massage

### DISCUSSION THREAD

**Discussion by shanshatil** [postalserver/postal#1872] [Help with using Postal]
Hello Experts

We are really new and don't have much experience. Yesterday we noticed that  981285 emails are in our queue. They are not successful.

![Screenshot_10](https://user-images.githubusercontent.com/1075759/154404816-1f1ce128-2750-4ffe-8dd0-0b79f8b228f9.jpg)

Anyone can help us how can we solve this issue.
That will be really helpful.

Thanks in advanced

**Comment by willpower232** [comment]
What is the output of `postal status`?
